change_notes/2024-01-12-fix-reported-fp-a3-9-1.md

```diff
@@ -0,0 +1,5 @@
+- `A3-9-1` - `VariableWidthIntegerTypesUsed.ql`:
+    - Exclude the plain char type. Still includes `signed char` and `unsigned char`.
```

Comment:
optional, similar comment to other PR , if you wanted to add the ref to the issue number resolved we could do that